src/swagger-specs/segmentation.yaml

```diff
@@ -3150,89 +3135,119 @@ components:
         labels:
           type: array
           items:
             type: string
           example: [core/C1]
+          description: Object-level data usage and attribute-based access control labels that are relevant to the audience.
         namespace:
           type: string
           example: AEPSegments
+          description: The namespace of the audience.
+          enum: [AAM, AAMSegments, AAMTraits, AEPSegments]
     ExternallyGeneratedAudience:
       type: object
       properties:
         id:
           type: string
           example: 32a83b5d-a118-4bd6-b3cb-3aee2f4c30a1
+          description: The ID of the audience.
         audienceId:
           type: string
           example: test-external-audience-id
+          description: The ID of the audience. For an externally-generated audience, this value is client-generated, and can be different than the `id` value.
         name:
           type: string
           example: externalSegment1
+          description: The name of the audience.
         namespace:
           type: string
           example: aam
+          description: The namespace of the audience.
+          enum: [AAM, AAMSegments, AAMTraits, AEPSegments]
         imsOrgId:
           type: string
           example: 1BD6382559DF0C130A49422D@AdobeOrg
+          description: The ID of the audience's organization.
         sandbox:
           type: object
           properties:
             sandboxId:
               type: string
               example: 6ed34f6f-fe21-4a30-934f-6ffe21fa3075
+              description: THe ID of the sandbox.
             sandboxName:
               type: string
               example: prod
+              description: The name of the sandbox.
             "type":
               type: string
               example: production
+              description: The type of the sandbox.
             default:
               type: boolean
               example: true
+              description: A boolean value that states whether or not this sandbox is the default sandbox.
+          description: Details about the sandbox the audience belongs to. More information about sandboxes can be found in the [sandbox overview]((https://adobe.com/go/sandbox-overview-en)).
         isSystem:
           type: boolean
           example: false
+          description: A property that shows if the audience is system-generated. If the audience is system-generated, it will not be included in any results by default.
         description:
           type: string
           example: "Last 30 days"
+          description: A description of the audience.
         type:
           type: string
           example: "ExternalSegment"
-        lifecyle:
+          description: A value that shows whether the audience is Platform-generated (`SegmentDefinition`) or externally-generated (`ExternalAudience`).
+          enum: [SegmentDefinition, ExternalAudience]
+        lifecycle:
```

Comment:
More repeated definitions. Let's rely on `$ref` more to avoid unnecessary bloat.